src/js/menu/menu.js

```diff
@@ -54,6 +65,72 @@ class Menu extends Component {
 
     return el;
   }
+
+  /**
+   * Handle key press for menu
+   *
+   * @param {Object} event Event object
+   * @method handleKeyPress
+   */
+  handleKeyPress (event) {
+    if (event.which === 37 || event.which === 40) { // Left and Down Arrows
+      event.preventDefault();
+      this.stepForward();
+    } else if (event.which === 38 || event.which === 39) { // Up and Right Arrows
```

Comment:
The https://github.com/timoxley/keycode library seems like it would work. Potentially a little overkill (I was thinking of something as simple as the `this.keys` array from http://oaa-accessibility.org/example/15/), but why reinvent the wheel??